doc/en/_templates/links.html

```diff
@@ -6,6 +6,6 @@
   <li><a href="https://github.com/pytest-dev/pytest/">pytest @ GitHub</a></li>
   <li><a href="http://plugincompat.herokuapp.com/">3rd party plugins</a></li>
   <li><a href="https://github.com/pytest-dev/pytest/issues">Issue Tracker</a></li>
-  <li><a href="http://pytest.org/latest/pytest.pdf">PDF Documentation</a>
```

Comment:
@RonnyPfannschmidt I agree that would be ideal, but as @The-Compiler said I also don't see how to do that in any automated manner.

FWIW this link was always pointing to the "latest" before moving to RTD, so unless we have a suggestion on how to easily update this link automatically I would rather have the broken link fixed now than later. :grin: